src/pages/sign-in/index.tsx

```diff
@@ -1,24 +1,39 @@
 import { Logo } from 'public/icons/nowruz/logo';
 import { useSelector } from 'react-redux';
-import { Navigate } from 'react-router-dom';
+import { Navigate, useLoaderData, useSearchParams } from 'react-router-dom';
+import { EventsRes } from 'src/core/api';
 import { IntroHeader } from 'src/modules/Auth/components/IntroHeader';
 import { SignInForm } from 'src/modules/Auth/containers/signin/SignInForm';
+import { generateEventsId } from 'src/modules/Events/constants';
+import TechSummit from 'src/modules/Events/TechSummit';
 import { RootState } from 'src/store';
 
 export const SignIn = () => {
+  const events = (useLoaderData() as EventsRes) || null;
+  const [searchParams] = useSearchParams();
+  const eventName = searchParams.get('event_name') || '';
   //TODO: remove this duplicate codes but works fine for now
   const status = useSelector((state: RootState) => state.identity.status);
   if (status === 'loading') return <div></div>;
   if (status === 'succeeded') return <Navigate to="/jobs" />;
+  const isTechSummit = eventName === 'tech4impact';
+  const eventsItems = events?.items || [];
 
   return (
-    <div className={`pt-12 px-4  md:pt-24 form-container`}>
-      <IntroHeader
-        title="Log in to your account"
-        description=" Welcome back! Please enter your details."
-        logo={<Logo width={48} height={48} />}
-      />
-      <SignInForm />
+    <div className="flex items-center h-screen">
+      <div className="flex-1">
+        <div className={`pt-12 px-4  md:pt-24 form-container`}>
+          <IntroHeader
+            title={isTechSummit ? 'Tech for Impact Summit 2024' : 'Log in to your account'}
```

Comment:
Is the title going to change for just `Tech summit event`? What about later events? Can't it be just the event name?